buildSrc/src/main/kotlin/Project.kt

```diff
@@ -5,6 +5,9 @@ object AndroidVersions {
   const val minSdkVersion_AndroidAuto = 23
   const val targetSdkVersion_AndroidAuto = 30
   const val compileSdkVersion_AndroidAuto = 31
+  const val minSdkVersion_Compose = 23
+  const val targetSdkVersion_Compose = 30
+  const val compileSdkVersion_Compose = 31
```

Comment:
:slightly_smiling_face:
nit:
```suggestion
  const val compileSdkVersion_Compose = 32
```

Reply:
```
Could not determine the dependencies of task ':compose-app:lintRelease'.
> Failed to install the following Android SDK packages as some licences have not been accepted.
     platforms;android-32 Android SDK Platform 32
  To build this project, accept the SDK license agreements and install the missing components using the Android Studio SDK Manager.
```
I don't see an urgent need to bump compile sdk version, we can do it the same time with the rest of the project(and also add the support in our CI docker image).